tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -130,3 +130,18 @@ stages:
         RunOnnxRuntimeTests: ${{ parameters.RunOnnxRuntimeTests }}
         # Liqun is fixing them
         RunStaticCodeAnalysis: true
+
+- stage: training_x64_debug_on_device_training
+  dependsOn: []
+  jobs:
+    - template: templates/win-cpu-ci.yml
+      parameters:
+        BuildConfig: 'Debug'
```

Comment:
So we prefer Debug build validation against Release?

Reply:
I just picked one of the two. I don't think we need to test both in the pipeline (at least for now). Can change to release.

Reply:
Debug is good for the purpose of this pipeline... we will add another packaging pipeline in future which will use release build.